4.0.0/migration.md

```diff
@@ -27,135 +146,148 @@ The migration process is initiated by the Installer if the following requirement
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 2/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 3/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 4/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 5/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 6/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 7/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 8/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 9/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 10/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 10/10)
+        deploy.go:1730] postgres-init-job failed
 
-     deploy.go:1711] removing postgres db init job
-     deploy.go:1714] running postgres db init job
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 1/10)
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 2/10)
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 3/10)
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 4/10)
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 5/10)
 
+1. You can stop the installer with `CTRL + c` because the migration will fail later in any case.
 
-1. Stop the installer  with CTRL + c because the migration will fail later in any case.
-2. Check which initialization failed. To do so, describe the pod related to the `postgres-init-job` job
+2. Check which initialization failed. To do so, describe the pod related to the `postgres-init-job` job:
 
-   `kubectl -n sysdigcloud describe pod -l "job-name=postgres-init-job"`
+    `kubectl -n sysdigcloud describe pod -l "job-name=postgres-init-job"`
 
-3. Find out which container in the pod is not terminated correctly by looking at State and Reason fields.
+3. Find out which container in the pod is not terminated correctly by looking at `State` and `Reason` fields.
    
    It should indicate that failed containers are one or more of the following: `profiling-worker-init`, `scanning-anchore-init`, `scanning-init`, `padvisor-init`, `sysdig-init` and `quartz-init`.
    
-4. Check the logs associated with the container that was failed to initialize:
+4. Check the logs associated with the container that has failed to complete:
 
     `kubectl -n sysdigcloud logs -l "job-name=mysql-latest-migrations" -c sysdig-init`
 
 
-### Latest MySQL Migration Job Failed
+### Latest MySQL Migrations Job Failed
 
-Stopping the MySQL migration job after 10 hours indicates that the `mysql-latest-migrations` job has not finished correctly.
+The whole migration process in this case will stop if after 10 hours the `mysql-latest-migrations` job has not finished correctly.
```

Comment:
The whole migration process might be terminated after 10 hours. This indicates that the `mysql-latest-migrations` job has not finished correctly.